packages/ui/src/ui/store/reducers/users/delete-user.ts

```diff
@@ -3,21 +3,28 @@ import {PayloadAction, createSlice} from '@reduxjs/toolkit';
 const initialState = {
     showModal: false,
     username: '',
```

Comment:
Do we need `showModal` and `username` here? May be we can use one field `usernameToDelete`?
It looks like we never want to display the dialog in case `username` is empty.